src/utils.jl

```diff
@@ -11,11 +11,12 @@ end
 
 function computep0(var)
     @extract var:W Z q
+    pc = 1/size(Z, 2)/10
```

Comment:
Why M=10 ? 
Is it something empirically working?

 Maybe setting a keyword argument pc_factor = 0.1? 
```suggestion
    pc = 1/(size(Z, 2)*pc_factor)
```